x-pack/plugins/serverless/public/navigation/navigation.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { type FC } from 'react';
+import {
+  Navigation,
+  NavigationKibanaProvider,
+  type NavigationProps,
+  type NavigationKibanaDependencies,
+} from '@kbn/shared-ux-chrome-navigation';
+
+export interface Props {
+  navProps: NavigationProps;
+  deps: NavigationKibanaDependencies;
+}
+
+export const SideNavigation: FC<Props> = ({ navProps, deps }) => {
+  return (
+    <NavigationKibanaProvider {...deps}>
+      <Navigation {...navProps} />
+    </NavigationKibanaProvider>
+  );
+};
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
I assume we're default exporting this to allow lazy loading (though I wonder what the value of lazy loading the side nav that's always there is anyway), but we should have a comment explaining why we're default exporting.